CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -272,107 +268,9 @@ public function fillTable(int $id, array $contactIDs, array $params, $legacyMode
     if (empty($tableQueries)) {
       return FALSE;
     }
+    $threshold = $ruleGroup->threshold;
```

Comment:
Where does this come from?

Reply:
@stesi561 further up

![image](https://github.com/user-attachments/assets/2076e59f-4d2b-470f-bb87-93f553c8fd46)

Reply:
Sorry - I wasn't clear. Looks like it's from here.
replacing  [$isInclusive, $isDie] = self::isQuerySetInclusive($tableQueries, $ruleGroup->threshold, $exclWeightSum);
with 
      [$isInclusive, $isDie] = self::isQuerySetInclusive($tableQueries, $threshold, $exclWeightSum);
passed in via the new function. Okay - Will try and circle back here later.